src/content/docs/style-guide/writing-docs/writer-workflow/tech-writer-workflow.mdx

```diff
@@ -55,72 +55,81 @@ Need to edit a doc? Use this table to decide where to work!
 
+Node and Yarn are tools used to build the site on your local machine.
+
 ### Build the site [#build]
 1. In your terminal, go to your cloned repo, **docs-website**. For example:
     - `cd ~/Documents/github/docs-website`
 2. Run yarn with the following commands: `yarn && yarn start`
 3. The site will take a few minutes to build. Make yourself some tea or coffee. 
-4. Once it's built, you can access your preview site in your browser by navigating to [http://localhost:8000/](http://localhost:8000/)
-
-## 3. Edit a doc [#edit-doc]
-Once your local environment and branch are set up, you're ready to edit a doc. Check out the style guide for writing guidelines.
-
-1. First, ensure your **Current Branch** in GitHub Desktop is set to the correct branch, not **Develop**.
-2. Navigate to the doc you want to edit in Finder. If I wanted to edit a Python agent doc, I would navigate to: `~/Documents/github/docs-website/src/content/docs/agents/python-agent/hosting-services/python-agent-stackato.mdx`
-3. Edit the doc in your text editor of choice. You should write docs in markdown language. Reference the style guide for help with formatting markdown
-4. Save the file with your edits, then follow the same process for any other docs you wish to edit.
-
-## 4. Commit your changes [#commit-changes]
-Once your edits are done, you can commit them. This stages your changes, which you will later push upstream to Github. By pushing your changes, everyone will have access to your branch and commits. 
-
-1. Navigate to GitHub Desktop. The left column should have a record of all the edits you have made to docs. 
-2. In the bottom left corner, name your commit and add a good description of your edits. It should be descriptive enough to ensure that someone can understand all the changes made by simply scanning this description.
-3. Click **Commit to [yourbranchname]**
+4. Once built, preview the site in your browser by navigating to [http://localhost:8000/](http://localhost:8000/)
 
 ## 5. Publish your commits [#publish-commits]
-
-Once you have committed your changes, you're almost ready to open your Pull Request.
-
-First, you need to ensure your branch is pushed upstream.
+Once you're satisfied with doc, you'll want to push all of your commits upstream. This makes them available to everyone else in the GitHub repository. 
 
 1. On GitHub desktop, click the blue **Publish Branch** button if available.
 2. If you don't see the Publish Branch, click the blue **Push Origin** button. 
 
-This will push all your commits upstream and make them available to everyone else through the GitHub repository.
+Now you're ready for a Pull Request. 
 
 ## 6. Open your pull request [#open-pr]
-Now that your commits are available to everyone, you need to notify people that your changes are ready to be merged into the `develop` branch.
+Your commits are available to everyone, but you'll need to notify people that your changes are ready to be merged into the `develop` branch.
```

Comment:
When a commit is pushed to a published branch, you can see that change on GitHub. 

Creating a PR is a way to communicate to the docs team that some work is ready to be reviewed for the purposes of merging it into `develop`. 

I feel like this doc highlights the ways in which we, as a team, haven't quite gotten the purpose of this process :)